cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -12,11 +14,15 @@
 {% if grains.cloud == 'gce' -%}
   {% if grains.cloud_config is defined -%}
     {% set cloud_config = "--cloud_config=" + grains.cloud_config -%}
+    {% set cloud_config_mount = "{\"name\": \"cloudconfigmount\",\"mountPath\": \"" + grains.cloud_config + "\", \"readOnly\": true}," -%}
```

Comment:
I rebased, and it appears somebody else came up with a nice way, so I repeated that here.